src/main/java/org/opensearch/timeseries/task/RealtimeTaskCache.java

```diff
@@ -21,7 +21,7 @@
  * 4. last job run time
  * 5. detector interval
  */
-public class ADRealtimeTaskCache {
+public class RealtimeTaskCache {
```

Comment:
Nit: may want to update above description to not be AD-specific

Reply:
yeah, updated